platform/android/java/build.gradle

```diff
@@ -270,19 +280,6 @@ task generateGodotMonoTemplates {
     finalizedBy 'zipGradleBuild'
 }
 
-/**
- * Generates the same output as generateGodotTemplates but with dev symbols
- */
-task generateDevTemplate {
-    // add parameter to set symbols to true
-    project.ext.doNotStrip = "true"
-
-    gradle.startParameter.excludedTaskNames += templateExcludedBuildTask()
-    dependsOn = generateBuildTasks("template")
-
-    finalizedBy 'zipGradleBuild'
-}
```

Comment:
Doesn't this removal mean that it's no longer possible to generate templates with debug symbols unless calling gradle from scons?

Reply:
Yes that's correct. Using scons, as @syntaxerror247 mentioned, will be the way to generate those templates.

This task also had the unforeseen side-effect of always enabling `doNotStrip` regardless of whether you were running it or not due to how gradle evaluates declared tasks.